packages/p2p-v2/src/pages/my-profile/screens/MyProfileCounterparties/MyProfileCounterpartiesTableRow/MyProfileCounterpartiesTableRow.tsx

```diff
@@ -7,11 +7,15 @@ import './MyProfileCounterpartiesTableRow.scss';
 
 type TMyProfileCounterpartiesTableRowProps = {
     id: string;
-    isBlocked: boolean;
+    is_blocked: boolean;
```

Comment:
```suggestion
    isBlocked: boolean;
```

Reply:
I have to leave this as is_blocked because of the data returned and is being spread to the component below. cause the return value is is_blocked not isBlocked. I could also pass it directly like this: `isBlocked={rowData.is_blocked} but then there were more eslint issues 😅 so I will leave it like this